src/send_kcidb.py

```diff
@@ -331,12 +331,12 @@ def _parse_test_node(self, origin, test_node):
         if artifacts:
             parsed_test_node['output_files'] = self._get_output_files(
                 artifacts=artifacts,
-                exclude_properties=('lava_log', 'test_log')
+                exclude_properties=('lava_log', 'job_txt')
```

Comment:
Is there any other type of job that logs to `test_log` or `job_txt`?

Would it make sense to align `kunit` to log to `test_log`?

Reply:
`kunit` nodes already have `test_log` file which has different content than job logs.

Reply:
I think that's a binary file. I need to check.

Reply:
The editor finds issues with `test_log` file formatting when I try to open it. There are some un-readable chars when I opened it with VSCode.
Could you please check and confirm which file should I use for log excerpt?

Reply:
As @nfraprado noted [on Slack](https://kernelci.slack.com/archives/C015N3UF7C1/p1718194687818789?thread_ts=1718186120.571499&cid=C015N3UF7C1) `test_log` would be the one to get an excerpt from because:

> [it] contains the KTAP output of the tests (including potential error messages and kernel error logs that are helpful to diagnose the error)

Reply:
Thanks for confirming. 
I'll update the code accordingly. I'll drop this commit from the PR as it's not relevant and open up a new one.